hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -16,7 +16,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "",
-  version:            "2.0.0",
+  version:            "1.1.0",
```

Comment:
Reducing to minor version change because the _late debug enablement_ got added in a backwards-compatible manner for what concerns SW